### PR TITLE
The saturated cover gets named, and near-zero flat does not

### LINKED ISSUE #197: A covered IR camera under an active emitter saturates flat (mean 255); the dark-gated diagnosis never sees the most common cover case
Found during #196's hardware validation on the ASUS 3277:0059 (built-in laptop camera), three runs, identical result each time.

## Measured

An opaque cover (card) over the camera with the emitter active does not produce a dark frame. It produces total saturation: the burst's chosen frame reads mean 255.0 with the camera classifying 10/10 frames, because the cover reflects the 850nm emitter straight back into the sensor at zero distance. The #196 diagnosis is gated on `mean < IR_DARK_HINT_MAX` (35), so no cause is ever named for the cover case the flat-frame arm was written for.

For contrast, the two related signatures already measured on this camera:

- E-shutter engaged (#186): firmware-substituted constant 144.0, stddev exactly 0.00, `privacy` reads 1, and capture refuses before diagnosis.
- Genuinely dark room with the emitter off (#196 phase C): mean 0.7-1.8 with real texture.

So on this hardware, "flat and saturated" is the cover signature, "flat at 144" is the shutter signature, and "dark with texture" is a dark scene. The current code names only causes below the dark gate.

## Shape

Extend the diagnosis entry to a second gate: a near-constant frame at ANY mean is not a scene. A flat frame at or near saturation with lit flags present reads as "something flat is covering the lens, reflecting the emitter back"; the existing `FlatFrame` wording rules apply (name the measurement, claim only what was measured on which camera). The `IRLUME_IR_EMITTER=off` output policy stays first.

Wants a stddev reading from a saturated covered frame before coding: 255-clipped frames are plausibly stddev 0.00 exactly, but that is an assumption until measured (the #196 harness logs only the mean today).

Relates to #185 (this is its seventh signature), #186, #192.

### PR DESCRIPTION
**Pull request by archledger**
Closes #197.

#196's hardware validation measured that the most common cover case is not dark: an opaque cover under the active emitter reflects it straight back and saturates the sensor. The dark-gated diagnosis never saw it. The follow-up measurement runs (instrumented `IRLUME_DEBUG_IR_FRAMES` now printing per-frame stddev, two runs per camera) put numbers on every band:

| observation | mean | stddev |
|---|---|---|
| ASUS covered, chosen frame | 255.0 | exactly 0.00 |
| NexiGo covered, chosen frame | 252.8 | 0.76 |
| covered mid auto-exposure walk-down | 245-254.7 | up to 3.63 (chosen), 49 (unchosen) |
| genuine strobe-off exposure, both cameras | 0.0 | 0.08-0.13 |
| real scenes (this hardware's records) | 38-168 | 35+ |

## What ships

- **`diagnose` is self-gating and answers `None` for scenes.** Two bands carry a diagnosis: dark (mean under 35) and saturated-flat (mean 250 up, spread 8 down). A saturated frame WITH texture is a close bright subject and stays undiagnosed. The call site's range check is a shortcut past the stddev pass on ordinary captures; the module's gates are authoritative.
- **`SaturatedFlat`**: named from the measured signature, with the lit-frame counts when the camera supplies them, and no firmware advice. The saturated floor (8) is deliberately wider than the dark flat floor (0.5): the NexiGo clips unevenly at 252.8/0.76, and "exactly constant" would have missed a fully covered camera there.
- **The dark `FlatFrame` arm is DELETED, not floored.** The first version kept it above a darkness floor; the review round showed the surviving 5-35 mean band has no measurement anywhere behind it (the strobe-off flats sit at 0.0, and the one measured dark-band blank, the shutter's 144.0, never reaches capture because the privacy control refuses at open), and my regression test had hidden the gap by standing a synthetic mean of 20 in for the measured 144. A dark flat frame now falls through to the emitter causes, where the advice is grounded; the shutter keeps its named cause through the privacy evidence; the saturated arm carries the only measured flat-cover signature.
- `IR_DARK_HINT_MAX` moves into `ir_dark` as the public `DARK_MEAN_MAX`, so the call-site shortcut and the module gate are one constant.
- Flat at MID brightness (the 144-style band) is deliberately undiagnosed: unmeasured territory, pinned by a test so the saturated floor cannot creep down to claim it.

## Review

One Codex round: one merge-blocking finding, taken in full (the dark-flat deletion above). Its "already correct" section confirmed the gate agreement, the off policy over both bands, and that every added test can fail on its asserted behavior.

## Evidence

- irlume-camera: 219 passed (7 in `ir_dark`, including the measured cover values verbatim as test rows, the off-silences-saturated policy, and the near-zero fall-through to grounded advice).
- Hand-applied mutants killed on a committed clean tree, re-proven on the final shape after the deletion: the saturated spread check dropped, the saturation floor lowered below measured scenes, and `off` no longer covering the saturated band.
- clippy, fmt and the doc lane clean locally.
- Hardware, ASUS 3277:0059, two interactive runs: **the covered phase now prints the saturated diagnosis** (mean 255, stddev 0.00, "with 5/10 frames marked illuminated", evidence named, no firmware advice), where the #196 build could only stay silent; `off` silent both runs, including captures at means 2.2-4.4; and `LitButDark` fired truthfully again in a genuinely dark room with nothing in range (chosen means 2-20, emitter firing). Run two was 4/4 green. The runs used the commit before the dark-flat deletion; the saturated arm those runs exercised is unchanged by it. NexiGo, two runs on the final sha (post-deletion build): 4/4 both, the saturated diagnosis firing at exactly its measured clip values (mean 253, stddev 0.76 - the test rows, live), phase A silent at working means 133-142, `off` silent at mean 0.0.

## Not tested

- ~~The BRIO's covered case~~ Now tested: three BRIO runs on the final build, every diagnosis assertion green - `NoEmitterDriven` with the ir-setup advice on its covered/dark phases (textured bleed, means 10-17), `off` silent three of three, and NO saturated line anywhere, which is the misfire this camera existed to rule out (its covered frames cannot saturate, having no emitter). Its pre-existing capture-probe failure (#192) appeared as before and gained a data point, recorded on that issue.
- Flat frames between the bands (mean 35-250) remain undiagnosed by design; no hardware here produces one (the ASUS 144 shutter blank refuses at open via the privacy control).

### CODE REVIEW

File: crates/irlume-camera/src/ir_dark.rs [hunk 1]
```diff
@@ -129,6 +199,22 @@ pub fn render(card: &str, mean: f64, cause: &IrDarkCause) -> Option<String> {
...
+             {mean:.0}, stddev {stddev:.2}){}. On both tested cameras this matched an \
+             opaque cover directly in front of the lens, bouncing the emitter straight \
+             back (covered measured 252.8-255.0; real scenes carry spread 35+). Remove \
+             whatever is in front of the camera.",
+            if *frames_lit > 0 {
+                format!(", with {frames_lit}/{frames_classified} frames marked illuminated")
+            } else {
+                String::new()
+            }
+        )),
```
[the remark this answers is not in the document]
Comment (archledger) [reply crates/irlume-camera/src/ir_dark.rs:205]: Taken in 6843475: the "bouncing the emitter straight back" clause now renders only when the camera marked frames illuminated; without that flag the message names the pattern and both candidate sources (cover, strong IR flooding the lens). A test pins each shape - the lit form must contain the mechanism, the unlit form must not.
